spec/controllers/world_controller_spec.rb

```diff
@@ -1,12 +1,30 @@
 RSpec.describe WorldController do
   describe "GET index" do
-    before do
-      stub_content_store_has_item("/world", GovukSchemas::Example.find("world_index", example_name: "world_index"))
+    context "without the GraphQL feature flag" do
+      before do
+        stub_content_store_has_item("/world", GovukSchemas::Example.find("world_index", example_name: "world_index"))
+      end
+
+      it "has a success response" do
+        get :index
+        expect(response).to have_http_status(:success)
+      end
     end
 
-    it "has a success response" do
-      get :index
-      expect(response).to have_http_status(:success)
+    context "with the GraphQL feature flag" do
+      before do
+        enable_graphql_feature_flag
+
+        stub_publishing_api_graphql_query(
+          Graphql::WorldIndexQuery.new("/world").query,
+          fetch_graphql_fixture("world_index"),
+        )
+      end
+
+      it "has a success response" do
+        get :index
+        expect(response).to have_http_status(:success)
+      end
```

Comment:
**suggestion:** assertions

Is there a way that this test can be expanded to make the test fixture loading and stubbing more meaningful? Maybe this is just the style in Collections/frontend controller tests, but it feels like a lot of quite specific setup just to test that a stubbed response has a status code of 200

Reply:
Agreed.  I added this because there was already a test like this, but we're already testing the exact same thing in the feature test.

I'm not sure how we'd make it any more useful though, so I'm inclined to say it should just be deleted.